libs/shared/src/main/java/io/crate/concurrent/limits/ConcurrencyLimit.java

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright 2018 Netflix, Inc.
```

Comment:
Is this the correct copyright header, should'nt we use ours? its already documented that this class is derived of netflix's one..